src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/ManagingOfficerIndividualDto.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDate;
+
+public class ManagingOfficerIndividualDto {
+
+    @JsonProperty("first_name")
+    private String firstName;
+
+    @JsonProperty("last_name")
+    private String lastName;
+
+    @JsonProperty("has_former_names")
+    private Boolean hasFormerNames;
+
+    @JsonProperty("former_names")
+    private String formerNames;
+
+    @JsonProperty("date_of_birth")
```

Comment:
For dates in the DTOs, do you not need a format annotation too? Something like:

@JsonFormat(pattern="yyyy-MM-dd")

? (pattern would need to match what we expect/is documented in the spec)

Have you tried an integration test with web client? Does it work without it?

Reply:
Right, but we do not need it right now :)

Reply:
Validation and data formatting will be soon integrated

Reply:
i looked at this the other day, the @JsonFormat(pattern="yyyy-MM-dd") seems only to apply when converting from json into string so i don't think we need to use it (our json is converted directly to LocalDate)

Reply:
So JSON data is ultimately just a string. I assume it uses the default pattern (which we happen to be using) to convert it. Otherwise, how would it work, right?

Reply:
i believe so, if we submit json that is not the correct format, it fails to convert to LocalDate and throws error

Reply:
Thanks @mwestacott and @markpit for the insight